index.js

```diff
@@ -74,6 +87,9 @@ const server = Bun.serve({
       clients = clients.filter((x) => x !== ws);
       console.log(`Disconnected with ${ws.remoteAddress}`);
     },
+    message(ws, code, reason) {
```

Comment:
Wrong parameters, should be `message(ws, message)`.

https://bun.sh/docs/api/http#reference

Reply:
Oops, that's fixed now.